playbook-website/app/javascript/site_styles/docs/_kit_doc.scss

```diff
@@ -153,3 +155,49 @@
 pre.highlight {
   padding: $space_xl;
 }
+
+/* VERTICAL SECTION SEPERATOR */
+
+.fixedHeightSSeparator {
+  height: 200px;
+  width:30px;
+  float:left;
+}
```

Comment:
You want to add a space between `width:` and `30px` on line 163. Same thing on line below.